prov/verbs/include/windows/verbs_nd.h

```diff
@@ -145,8 +145,38 @@ struct nd_event_base {
 	OVERLAPPED ov;
 	nd_event_handler event_cb;
 	nd_error_handler error_cb;
+	ofi_mutex_t lock;
+    pthread_cond_t cond;
+	uint32_t cb_pending;
 };
 
+static inline HRESULT cancel_pending(struct nd_event_base *event,
```

Comment:
This should just be a standard call in some source file, no need for static inline.  Also, please try to add a prefix to all calls/structures/globals related to this provider